.internal-ci/test/fs-integration/test_config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Just a question.  Did you intend this to be an example template?  I see that this file is listed in the .gitignore